app/index.js

```diff
@@ -33,10 +37,7 @@ server.on('listening', onListening);
 /* Proxy websockets */
 
 var proxy = httpProxy.createProxyServer({
-  target: {
-      host: process.env.SHINY_HOST,
-      port: process.env.SHINY_PORT
-    }
+  target: config.shiny,
 });
```

Comment:
Does this make sense? Either `SHINY_HOST` and `SHINY_PORT` are named incorrectly (should they be `KIBANA_HOST` and `KIBANA_PORT`?), or is this websockets proxy not needed? Does Kibana use websockets? Presumably they don't work anyway, because of ELB?

Reply:
Why are there 2 proxy servers (one here and one in `routes.js`)?

Reply:
Yes, see: https://github.com/ministryofjustice/analytics-platform-kibana-auth-proxy/pull/1#discussion_r174488885

I'll try to tidy up further.

Reply:
https://github.com/ministryofjustice/analytics-platform-kibana-auth-proxy/pull/1/commits/f43d8380c7845af9e43cc7e61987d13a1501539d